src/auto-evo/AutoEvoRun.cs

```diff
@@ -619,4 +624,67 @@ private void RunSingleStepToCompletion(IRunStep step)
         // Doing the steps counting this way is slightly faster than an increment after each step
         Interlocked.Add(ref completeSteps, steps);
     }
+
+    private void UpdateMap(bool playerCantGoExtinct)
+    {
+        Parameters.World.Map.UpdateGlobalTimePeriod(Parameters.World.TotalPassedTime);
+
+        // Update populations before recording conditions - should not affect per-patch population
+        Parameters.World.Map.UpdateGlobalPopulations();
+
+        // Needs to be before the remove extinct species call, so that extinct species could still be stored
+        // for reference in patch history (e.g. displaying it as zero on the species population chart)
+        foreach (var entry in Parameters.World.Map.Patches)
+        {
+            entry.Value.RecordSnapshot(true);
+        }
+
+        var extinct = Parameters.World.Map.RemoveExtinctSpecies(playerCantGoExtinct);
+
+        foreach (var species in extinct)
+        {
+            Parameters.World.RemoveSpecies(species);
+
+            GD.Print("Species ", species.FormattedName, " has gone extinct from the world.");
```

Comment:
I'd remove this logging, timeline should include this info already or be extended to include events like this. I plan on removing all GD.Prints coming from auto-evo as I see it now as a bit of a problematic thing, so it'd help me if this could be removed in this PR.

Reply:
I want to at least comment all those prints too ...